### PR TITLE
Update accessibility elements

### PR DESCRIPTION
**Pull request by pprabesh**
This PR adds some accessibility attribute use cases in the RIch content.

### CODE REVIEW

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md [hunk 1]
```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
...
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. 
+Some of the widely used attributes are listed below. For the entire list of supported attributes, please refer to [this list](https://github.com/LivePersonInc/json-pollock/blob/master/js/schema/accessibilityWeb.json).
+
+#### Properties
+
+| Property Name    | Description                                                                 |
+| :--------------- | :-------------------------------------------------------------------------- |
+| role             | Describes the role of an element for screen readers                         |
```
Comment (j9t) [inline pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:573]: Nit: Consider marking the attribute names up as code.

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md [hunk 2]
```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
...
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. 
+Some of the widely used attributes are listed below. For the entire list of supported attributes, please refer to [this list](https://github.com/LivePersonInc/json-pollock/blob/master/js/schema/accessibilityWeb.json).
+
+#### Properties
+
+| Property Name    | Description                                                                 |
```
Comment (j9t) [inline pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:571]: Please switch to sentence case (“Property name”).

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md [hunk 3]
```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them.
```
Comment (j9t) [inline pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:566]: Please add another line if you want two paragraphs, or remove the new line to make this one paragraph :)

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md [hunk 4]
```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
```
Comment (j9t) [inline pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:564]: Please switch to sentence case.

**Review by j9t — CHANGES_REQUESTED** [review]
Also: In the front matter, add `date_updated` to indicate a substantial update.

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md (hunk 2 above)
Comment (pprabesh) [reply pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:571]: @j9t I wanted it to be consistent with other elements at other places in the page so I put a capital N.
https://developers.liveperson.com/getting-started-with-rich-messaging-introduction.html#element-styling

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md (hunk 4 above)
Comment (pprabesh) [reply pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:564]: @j9t same with the sentence case. I see non sentence case in other places too. So I thought this was the practice.

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md (hunk 1 above)
Comment (pprabesh) [reply pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:573]: I was just following how the attributes were addressed in other parts in the page. And added the code as an example below.

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md (hunk 4 above)
Comment (j9t) [reply pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:564]: Fair enough—we can work with this for now. It’s still widespread on the site, but we are switching to sentence case; see https://github.com/LivePersonInc/developer-center#notes-on-content-and-code for details.

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md [hunk 5]
```diff
@@ -560,4 +561,48 @@ Each basic element can have specific style rules defined for it, controlling how
...
+        "aria-level": "3",
+        "tabindex":"1",
+        "aria-label":"This is aria-label for button",
+        "aria-hidden":"false"
+      }
+  }
+}
+```
+
+Date Updated: 10/24/2024
```
Comment (j9t) [inline pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md:606]: @pprabesh, please use a front matter entry for this information: look for “date_updated” [in the docs](https://github.com/LivePersonInc/developer-center#updating-and-creating-headers) ([example](https://github.com/LivePersonInc/developer-center/blame/master/pages/Documents/APIsandSDKs/overview.md#L13)).

**Review by j9t — CHANGES_REQUESTED** [review]

File: pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md (hunk 5 above)
Comment (pprabesh) [reply pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md]: @j9t Sorry, I misunderstood earlier. Updated the front matter.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
Thanks @pprabesh! LGTM. Please merge or confirm so that we merge.

**Comment by pprabesh** [comment]
@j9t I dont think I have the necessary permissions to merge. Please merge when possible.
Thank you!

**Comment by j9t** [comment]
Thanks @lkisoslive :)

**Pull request merged**